tests/test_packages/unit/test_source_suppress.py

```diff
@@ -88,7 +88,7 @@ def test_one_liner_some_checkers(self):
             sp_handler.suppress_comment(), 'some really really long comment')
 
     def test_multi_liner_some_checkers(self):
-        """Doc."""
+        """There is suppress comment above the bug line."""

```

Comment:
A more specific comment would be better like:
'Tests if there is a multi line suppress comment with multiple checkers in it'